django_perf_rec/api.py

```diff
@@ -111,11 +113,17 @@ def load_recordings(self):
     def save_or_assert(self):
         orig_record = self.records_file.get(self.record_name, None)
 
+        if settings.MODE == 'none':
+            assert orig_record is not None, "Performance record does not exist for {}".format(self.record_name)
```

Comment:
The message needs to be something like 'original performance record does not exist for ...'